src/users/users.service.ts

```diff
@@ -26,4 +31,45 @@ export class UsersService {
     async getProfile(userId: number, companyId: number): Promise<any | undefined> {
         return this.usersRepository.findOne( { relations: ["manager"], where: { employeeId: userId, companyId: companyId } } );
     }
+
+    async removeUser(employeeId: number, companyId: number): Promise<DeleteResult> {
+        const user = await this.usersRepository.findOne({ employeeId: employeeId, companyId: companyId })
+        await this.loginRepo.delete({employee: user});
+        return await this.usersRepository.delete(user);
+    }
+    
+    async createUser(createuserDto: Users & Login): Promise<Users> {
+        
+        const user = new Users();
+        // user.company = createuserDto.company;
+        let company = await this.companyRepository.findOne()
+        if (!company ) {
+            company = await this.companyRepository.save({companyId: 1, name: 'Bennedict Scrumberbatch'})
+        }
+        user.company = company
+        // const store = await this.usersRepository.find();
+        
+        // user.employeeId = store.length + 1;
+        user.employeeId = createuserDto.employeeId;
+        user.companyId = createuserDto.companyId;
+
+        user.firstName = createuserDto.firstName;
+        user.lastName = createuserDto.lastName;
+
+        user.isManager = Boolean(createuserDto.isManager);
+        user.positionTitle = createuserDto.positionTitle;
+        user.startDate = new Date(createuserDto.startDate);
+        
+        // if (store.length > 0) 
+        //     user.manager = await this.usersRepository.findOne({where:{employeeId : 1}}) // set employeeId 1 as manager for now
+        user.manager = createuserDto.manager
```

Comment:
I believe by the employee id as this was the format given in the sample data, though now that you mention it I'm not sure how that'll work with multiple companies the way its set right now. We can address that down the line in another ticket when we get there. I believe typeorm can also extract the appropriate ids from an object format so that may help

Reply:
I take this back - currently manager relations aren't being set on create. I'll create a task in trello on our backlog to make a not to come back and fix this.